BE/src/main/java/team07/airbnb/controller/AccommodationController.java

```diff
@@ -51,6 +54,70 @@ public AccommodationListResponse createAccommodation(@RequestBody AccommodationC
         ));
     }
 
+    @Tag(name = "Host")
+    @Operation(summary = "숙소의 방 정보 수정")
+    @Authenticated(Role.HOST)
+    @PatchMapping("{id}/roomInfo")
+    @ResponseStatus(HttpStatus.OK)
+    public AccommodationDetailResponse updateRoomInformation(@PathVariable Long id,
+                                                             @RequestBody RoomInformation info,
+                                                             TokenUserInfo user) {
+        return AccommodationDetailResponse.of(accommodationService.updateAccommodation(
+                id,
+                info,
+                user.id()
+        ));
```

Comment:
```suggestion
        return AccommodationDetailResponse.of(
            accommodationService.updateAccommodation(
                id,
                info,
                user.id()
            )
        );
```

이건 제안이고 스타일에 따라서 취사선택해주시면 좋을꺼같은데요, 결국             accommodationService.updateAccommodation() 를 호출한다는걸 조금더 잘 보이게 하고싶어서 저는 개행을 넣어주는편이에요